groupsforsections.py

```diff
@@ -0,0 +1,85 @@
+import json
+import sys
+import logging
+import requests
+import urllib
+import utils
+import yaml
+
+
+def create_group_category():
+    logging.debug('create_group_category() Called')
+
+    url = canvas_url + '/api/v1/courses/' + course_id + '/group_categories?name=' + urllib.quote_plus(
+            group_category_name)
+    response = api_handler(url, 'Post')
+    if response is None:
+        logging.error('Api call for \'creating group categories\' is not successful')
+        sys.exit(1)
+
+    if response.status_code != requests.codes.ok:
+        error_msg_handler(response)
+        sys.exit(1)
+    group_category = json.loads(response.text)
+    return group_category['id']
+
+
+def api_handler(url, request_type):
+    logging.debug('api_handler() called')
+    logging.info('URL: ' + url)
+    response = None
+    headers = {'content-type': 'application/json', 'Authorization': 'Bearer' + ' ' + canvas_token}
```

Comment:
Are the header names and MIME types already available as constants in Python?  If not, define them.  (Also, a minor thing, but `content-type` should be capitalized as `Content-Type`.)

The additional concatenation of `'Bearer' + ' '` isn't necessary.